src/games/strategy/triplea/ui/MapRouteDrawer.java

```diff
@@ -129,9 +134,14 @@ private void drawCustomCursor(Graphics2D graphics, RouteDescription routeDescrip
    * @param xOffset The horizontal pixel-difference between the frame and the Map
    * @param yOffset The vertical pixel-difference between the frame and the Map
    * @param jointsize The diameter of the Points being drawn
+   * @param width The width of the Map
+   * @param height The height of the Map
    * @param scale The scale-factor of the Map
    */
-  private void drawDirectPath(Graphics2D graphics, Point start, Point end, int xOffset, int yOffset, double scale) {
+  private void drawDirectPath(Graphics2D graphics, Point start, Point end, int xOffset, int yOffset, double scale,
+      int width, int height) {
+    start = getPointOnMap(start, xOffset, yOffset, width, height);
```

Comment:
@DanVanAtta  looks messy then - changed in in a seperate commit in order to easily revert this...